src/pages/admin/AdminMainIndex.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { useQuery } from '@tanstack/react-query';
+import { getMyInfo } from 'apis/userInfo';
+import { AdminNoGroupPage, AdminNoMemberPage } from 'pages/admin/ETCMainPage';
+import SchedulePage from 'pages/SchedulePage';
+import { UserData } from 'apis/types';
+
+const AdminMainIndex = (): JSX.Element => {
+  const { data: membersData } = useQuery(['getMyInfo'], getMyInfo, { suspense: true });
+  const hasGroup = membersData?.data.groupName !== null;
+  const hasMember = membersData && membersData.data.members.length > 1;
+
+  return (
+    <>
+      {!hasGroup && <AdminNoGroupPage />}
+      {hasGroup && !hasMember && <AdminNoMemberPage />}
+      {hasGroup && hasMember && <SchedulePage members={membersData?.data.members as UserData[]} />}
```

Comment:
membersData가 undefined가 실제로 될 수 있는 것이기 때문에 타입에러가 나는게 정상입니다
따라서 as로 강제로 타입을 변경하는 것은 좋지 않고, undefined인 경우에 대응하셔야 합니다!

```suggestion
      {hasGroup && hasMember && <SchedulePage members={membersData?.data.members || []} />}
```
이렇게 undefined인 경우엔 빈배열을 줄 수 있을 것 같습니다!